core/modules/core/src/com/haulmont/yarg/formatters/impl/doc/connector/OOServer.java

```diff
@@ -183,13 +183,13 @@ public void run() {
 
         ArrayList<String> options = new ArrayList<String>();
 
-        options.add("-nologo");
-        options.add("-nodefault");
-        options.add("-norestore");
-        options.add("-nocrashreport");
-        options.add("-nolockcheck");
-        options.add("-nofirststartwizard");
-        options.add("-headless");
+        options.add("--nologo");
+        options.add("--nodefault");
+        options.add("--norestore");
+        options.add("--nocrashreport");
+        options.add("--nolockcheck");
+        options.add("--nofirststartwizard");
+        options.add("--headless");
 
```

Comment:
We can't change the format of the command-line argument, because YARG can be used with old OpenOffice 3.1.0. OpenOffice 3.1.0 supports command line arguments only with one `-` (https://wiki.openoffice.org/wiki/Documentation/OOo3_User_Guides/Getting_Started/Starting_OpenOffice.org)